hr_infraction/security/ir.model.access.csv

```diff
@@ -1,12 +1,6 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
Please addapt the acces right with the base groups, no need to leave the employee to delete his infraction...